vllm/entrypoints/openai/protocol.py

```diff
@@ -1034,7 +1089,8 @@ class ScoreRequest(OpenAIBaseModel):
         description=(
             "The priority of the request (lower means earlier handling; "
             "default: 0). Any priority other than 0 will raise an error "
-            "if the served model does not use priority scheduling."))
+            "if the served model does not use priority scheduling."),
+    )
```

Comment:
there's a lot of formatting changes in this PR, is it just about pre-commit being updated? If that's the case a rebase would help clearing a lot of that out.

Reply:
True, I am not sure what caused that and will look into that!

Reply:
I checked and my pre-commit is up to date and runs fine with `pre-commit run --all-files`. My guess is that my default VSCode formatter. If needed I will manually revert these changes!

Reply:
@hmellor does this usually happen..?